bin/genesis

```diff
@@ -771,12 +787,12 @@ sub __prompt_for_line {
 			$validate = sub() {
 				my $needle=shift;
 				my @matches = grep {$_ eq $needle} @__vlist;
-				return (scalar(@matches) != 0 ? "" : ($_[1] ? $_[1] : "Expecting one of ".join(", ",@__vlist)))
+				return (scalar(@matches) != 0 ? "" : ($_[1] ? $_[1] : "Expecting one of ".join(", ",@__vlist)));
 			}
 		} elsif ($validation eq "vault_path") {
-			my $validate = sub() {
+			$validate = sub() {
 				system "safe exists $_[0]";
-				return ( $? eq 0 ? "" : ($_[1] ? $_[1] :"$_[0] not found in vault\n"))
+				return ( $? eq 0) ? "" : ($_[1] ? $_[1] :"$_[0] not found in vault\n");
```

Comment:
These appear to be unrelated bug fixes, and should really go on their own PR.

Reply:
this function `__prompt_for_line` is called by `genesis ci` command, so I just put those bug fix in the same PR. I did put those bug fix in its own commit to separate from the main code for adding `ci` feature.